### DISCUSSION TITLE
Why constraints show on some flights but not others?

### DISCUSSION THREAD

**Discussion by robinottawa** [flybywiresim/aircraft#3318] [Q&A]
I don't see a pattern to the constraints showing on the FP map (on the panel - not sure of the name) and on the MCDU. Sometimes I get more constraints and sometimes less/fewer. Is this due to FBW developments or just the MSFS flight plans database?
Thanks, Rob

**Comment by ChruutvoLuzi** [comment]
Depends if there are any constraints on your Arrival/Departure. Best way to find if constraints are correct, is to consult your local chart. I found them to be accurate on Arrivals/Departures most of the time.
